bucket/neovim-bare-nightly.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "0.7.0-dev-812-g2f31e7b88",
+    "description": "Vim fork focused on extensibility and usability",
+    "homepage": "https://neovim.io",
+    "license": {
+        "identifier": "Apache-2.0,Vim",
+        "url": "https://github.com/neovim/neovim/blob/master/LICENSE"
+    },
+    "suggest": {
+        "vcredist": "extras/vcredist2015"
```

```suggestion
        "vcredist": "extras/vcredist2022"
```
VCRedist 2022 covers the functionality of VCRedist 2015-2022. See https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170